exercises/rna-transcription/example.rs

```diff
@@ -1,4 +1,4 @@
-/// The possible nucleotides in DNA and RNA
+/// The possible nucleotides in Dna and Rna
```

Comment:
Comments are exceptions; normal acronym rules (all-caps) should apply there.

Reply:
Addressed in 64b1b68. Thanks for the review!